src/core/mac/mac.cpp

```diff
@@ -592,6 +612,12 @@ void Mac::PerformOperation(void)
         mOperation = kOperationWaitingForData;
         RadioReceive(mChannel);
     }
+    else if (mPendingTransmitOobFrame)
+    {
+        mPendingTransmitOobFrame = false;
+        mOperation = kOperationTransmitOutOfBandFrame;
+        StartCsmaBackoff();
```

Comment:
Is there an option to transmit OOB without CSMA?

Reply:
Not currently. As of now there is no need.  This feature full-fills a specific need by Nest.